src/docs/internal/platforms/index.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Platform Portals
+noindex: true
+---
+
+Platform content lives in `src/platforms` and follows some specific rules to generate content. The content is rendered using the same features as all other [pages](../), but has some additional functionality exposed to clone content and encapsulate platforms as portals.
+
```

Comment:
Perhaps add in:

Platform content is hierarchical. When common content is present, it will be used only if the platform or the child of a primary platform has not provided an override. In sum, guide content is used when present. If not present, platform/common content is used. If not present, /common content is used.

Reply:
imo this probably needs a larger follow up, as i think adding it here isnt in depth enough